packages/webapp/src/components/Animals/AddAnimalsFormCard/styles.module.scss

```diff
@@ -0,0 +1,35 @@
+.form {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+}
+
+.formHeader {
+  display: flex;
+  justify-content: space-between;
+}
+
+.removeBtn {
```

Comment:
It looks like this button has active or hover state: https://www.figma.com/file/eMffC0h3JPlWtzNttxJpil/Litefarm-Updated?node-id=3881-23342&mode=dev